### DISCUSSION TITLE
Question about TorchVisionFC model

### DISCUSSION THREAD

**Discussion by ctestagrose** [Project-MONAI/MONAI#4865] [Q&A]
I am interested in using the TorchVisionFC model. I see that the default model is resnet. Is there any way to use another torchvision model that is specified https://pytorch.org/vision/stable/models.html ? I would like to replace resnet with inception v3. 

When I replace resnet with inception v3 I am receiving the following error:

```
RuntimeError: Expected 4-dimensional input for 4-dimensional weight [192, 768, 1, 1], but got 2-dimensional input of size [16, 1000] instead
```
I am thinking this is an issue with either the way I am loading the data or possibly a misinterpretation on my part in regard to the way the TorchVisionFC model works.

Thank you, I am extremely appreciative of any help.

**Comment by wyli** [comment]
looks like you are having a batch size 16 and 1000-dimensional vector as the input to a torchvision FC model. the shape looks more like an output of a classification model to me. there are a few use cases in the unit tests, in case one of them matches your use case.. https://github.com/Project-MONAI/MONAI/blob/1516ca758090dd8ed2890b64554a77f681a0e224/tests/test_torchvision_fc_model.py#L26
